### PR TITLE
Add report generation function to master branch

### PR DESCRIPTION
**Pull request by Yujian-Fu**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/FilterDangerCommand.java [hunk 1]
```diff
@@ -43,9 +43,10 @@ public BluetoothSummaryMessage execute(AppStorage dao) {
         Conditions cond = new DangerConditions(this.THRESHOLD);
         Aggregators<BluetoothPings, BluetoothPingsSummary> agg = new GroupByIDPairsAggregators();
         ArrayList resp  = dao.search(cond, agg);
+        System.out.println("Searching danger");
+
```
Comment (JKOK005) [inline src/main/java/seedu/address/logic/commands/FilterDangerCommand.java]: Please use logger. Don't use println

File: src/main/java/seedu/address/logic/commands/FilterUserIDCommand.java [hunk 2]
```diff
@@ -37,6 +37,7 @@ public FilterUserIDCommand validate(String arguments) throws ParseException {
     public BluetoothPingsMessage execute(AppStorage dao) {
         Conditions cond = new UserIDConditions(USER_ID);
         ArrayList resp  = dao.search(cond);
+        System.out.println("Search id");
```
Comment (JKOK005) [inline src/main/java/seedu/address/logic/commands/FilterUserIDCommand.java]: Use logging

**Review by JKOK005 — CHANGES_REQUESTED** [review]
2 issues:

a) Do not include *.jar / *.zip files in the PR. Either delete them or blacklist them in .gitignore.

b) Do not use system.our.println. Use the logging module

**Review by JKOK005 — CHANGES_REQUESTED** [review]
Use Camel casing (https://simple.wikipedia.org/wiki/CamelCase) to name modules. 

What you are doing is a_c_b. 
It should be AbC.

File: build.gradle [hunk 3]
```diff
@@ -23,6 +23,7 @@ targetCompatibility = JavaVersion.VERSION_11
 repositories {
     mavenCentral()
     maven { url 'https://oss.sonatype.org/content/repositories/snapshots/' }
+    maven { url 'https://mvnrepository.com/artifact/com.itextpdf/kernel/7.0.4'}
```
Comment (JKOK005) [inline build.gradle]: Remove this

File: build.gradle [hunk 4]
```diff
@@ -60,10 +61,13 @@ dependencies {
 
     implementation group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.7.0'
     implementation group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jsr310', version: '2.7.4'
+    implementation group: 'com.itextpdf', name: 'kernel', version: '7.0.4'
+    implementation group: 'com.itextpdf', name: 'layout', version: '7.1.9'
 
     testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-api', version: jUnitVersion
 
     testRuntimeOnly group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: jUnitVersion
+    compile fileTree(dir:'libs',include:['*.jar'])
```
Comment (JKOK005) [inline build.gradle]: Remove this

Comment (Yujian-Fu) [reply build.gradle]: Already removed.

**Review by JKOK005 — APPROVED** [review]

**Pull request merged**